verticapy/tests/stats/test_stats.py

```diff
@@ -0,0 +1,153 @@
+# (c) Copyright [2018-2020] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest, warnings
+from verticapy import drop_table, set_option, str_sql
+import verticapy.stats as st
+
+set_option("print_info", False)
+
+
+@pytest.fixture(scope="module")
+def titanic_vd(base):
+    from verticapy.learn.datasets import load_titanic
+
+    titanic = load_titanic(cursor=base.cursor)
+    yield titanic
+    with warnings.catch_warnings(record=True) as w:
+        drop_table(
+            name="public.titanic", cursor=base.cursor,
+        )
+
+
+class TestStats:
+    def test_abs(self, titanic_vd):
+        assert st.abs(titanic_vd["age"]) == 'ABS("age")'
```

Comment:
I don't understand these tests. Does applying the new functions on a column of vDataFrame return a text?

Reply:
it returns an str_sql that when it is compare to str, it will be casted during the comparison.